clouddriver-google/src/test/groovy/com/netflix/spinnaker/clouddriver/google/deploy/converters/UpsertGoogleLoadBalancerAtomicOperationConverterUnitSpec.groovy

```diff
@@ -81,4 +88,79 @@ class UpsertGoogleLoadBalancerAtomicOperationConverterUnitSpec extends Specifica
     then:
       operation instanceof UpsertGoogleLoadBalancerAtomicOperation
   }
+
+  void "createGoogleHttpLoadBalancerDescription type returns UpsertGoogleLoadBalancerDescription and CreateGoogleHttpLoadBalancerAtomicOperation"() {
+    setup:
+    def hc = [
+      "name"              : "basic-check",
+      "requestPath"       : "/",
+      "port"              : 80,
+      "checkIntervalSec"  : 1,
+      "timeoutSec"        : 1,
+      "healthyThreshold"  : 1,
+      "unhealthyThreshold": 1
+    ]
+    def input = [
+      "loadBalancerType"      : "HTTP",
+      "credentials"           : "my-google-account",
+      "googleHttpLoadBalancer": [
+        "name"          : LOAD_BALANCER_NAME,
+        "portRange"     : PORT_RANGE,
+        "defaultService": [
+          "name"       : DEFAULT_SERVICE,
+          "backends"   : [],
+          "healthCheck": hc,
+        ],
+        "certificate"   : "",
+        "hostRules"     : [
+          [
+            "hostPatterns": [
+              "host1.com",
+              "host2.com"
+            ],
+            "pathMatcher" : [
+              "pathRules"     : [
+                [
+                  "paths"         : [
+                    "/path",
+                    "/path2/more"
+                  ],
+                  "backendService": [
+                    "name"       : PM_SERVICE,
+                    "backends"   : [],
+                    "healthCheck": hc,
+                  ]
+                ]
+              ],
+              "defaultService": [
+                "name"       : DEFAULT_PM_SERVICE,
+                "backends"   : [],
+                "healthCheck": hc,
+              ]
+            ]
+          ]
+        ]
+      ]
+    ]
+
+    when:
+    def description = converter.convertDescription(input)
```

Comment:
I think all other google unit tests indent after the `when:` and `then:` style blocks